File: interface-vs-implementation/interface-vs-implementation.java
public class MyClass {
  private String first, second;

  public void set(String x, String y) {
    first = x;
    second = y;
  }

  public String getFirst() {
    return first;
  }

  public String getSecond() {
    return second;
  }

  public static void main(String[] args) {
    MyClass objectA = new MyClass();

    objectA.set("Hello, ", "World!");

    System.out.println(objectA.getFirst() + objectA.getSecond());
  }
}

File: interface-vs-implementation/interface-vs-implementation.js
// chatGPT 3.5 fixed
class MyClass {

	constructor() {
		this.first = "";
		this.second = "";
	}

	set(f, s){
		this.first = f;
		this.second = s;
	}

	getFirst() {
		return this.first;
	}

	getSecond() {
		return this.second;
	}
}


function main() {

	const objectA = new MyClass();

	objectA.set("Hello, ", "World!");

	console.log( objectA.getFirst() + objectA.getSecond(), "\n" )

	return null;
}

main();
